routes/web.php

```diff
@@ -334,6 +335,8 @@
                         Route::get('', [PostController::class, 'show'])->name('post.show');
                         Route::get('edit', [PostController::class, 'edit'])->name('post.edit');
                         Route::put('update', [PostController::class, 'update'])->name('post.update');
+                        Route::put('/photos/upload', [PostPhotoController::class, 'update'])->name('post.photos.upload');
```

Comment:
This particular route is not really and update, nor should be a `put` route, because you don't update an existing photo object. And you named the route with `upload`, which is relevant.
The controller method should be a `create` and the route should be a `post`, right?

Reply:
Damn you are right.

Reply:
done